homeassistant/components/fan/alfawise.py

```diff
@@ -0,0 +1,95 @@
+"""
+Alfawise fan platform that has a fake fan.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/fan.alfawise/
+"""
+import logging
+
+from homeassistant.components.fan import (SPEED_LOW, SPEED_HIGH,
+                                          FanEntity, SUPPORT_SET_SPEED)
+from homeassistant.const import CONF_DEVICES, CONF_NAME, CONF_MAC
+from homeassistant.const import STATE_OFF
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['pyAlfawise==0.4-beta']
+
+
```

Comment:
Configuration validation is missing -> https://www.home-assistant.io/developers/code_review_platform/#3-configuration